### PR TITLE
sentencepiece: add spec file

### PR DESCRIPTION
**Pull request by kenhys**

### CODE REVIEW

File: sentencepiece/yum/sentencepiece.spec [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+%prep
+%setup -q -n %{name}-%{version}-Source
+
+%build
+%cmake3 . -DCMAKE_INSTALL_LIBDIR=lib64
+make %{?_smp_mflags}
+
+%install
+rm -rf $RPM_BUILD_ROOT
+make install DESTDIR=$RPM_BUILD_ROOT INSTALL="install -p"
```
Comment (kou) [inline sentencepiece/yum/sentencepiece.spec]: Why is `INSTALL` needed?

File: sentencepiece/yum/sentencepiece.spec [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+rm -f $RPM_BUILD_ROOT%{_libdir}/libsentencepiece.a
+rm -f $RPM_BUILD_ROOT%{_libdir}/libsentencepiece_train.a
```
Comment (kou) [inline sentencepiece/yum/sentencepiece.spec]: Can we use `rm -f $RPM_BUILD_ROOT%{_libdir}/*.a` here?

File: sentencepiece/yum/sentencepiece.spec [hunk 3]
```diff
@@ -0,0 +1,85 @@
...
+Requires:	%{name}-libs = %{version}-%{release}
+
+%description devel
+This package contains header files to develop a software using SentencePiece
+
+%prep
+%setup -q -n %{name}-%{version}-Source
+
+%build
+%cmake3 . -DCMAKE_INSTALL_LIBDIR=lib64
```
Comment (kou) [inline sentencepiece/yum/sentencepiece.spec]: Should we use `%{_libdir}` instead of `lib64` here?

Comment (kenhys) [reply sentencepiece/yum/sentencepiece.spec]: Thanks, I've fixed it.

File: sentencepiece/yum/sentencepiece.spec (hunk 2 above)
Comment (kenhys) [reply sentencepiece/yum/sentencepiece.spec]: Thanks, I've fixed to unify it.

File: sentencepiece/yum/sentencepiece.spec (hunk 1 above)
Comment (kenhys) [reply sentencepiece/yum/sentencepiece.spec]: Thanks, it is no need to do it, I've removed.

**Pull request merged**